core/scripts/config_generate.py

```
@@ -139,14 +140,11 @@ def zip_language_folders(config_store_path, config_store_packed_path):
     config_store_packed_path = os.path.join(assets_path, 'config_store_packed')
     if os.path.exists(config_store_path):
         shutil.move(config_store_path, config_store_path + '_bak')
-    if not os.path.exists(config_store_path):
-        os.makedirs(config_store_path)
-    if not os.path.exists(config_store_packed_path):
-        os.makedirs(config_store_packed_path)
+    os.makedirs(config_store_path, exist_ok=True)
+    os.makedirs(config_store_packed_path, exist_ok=True)
     for lang in lang_list:
```

Ruff (0.8.0)

`lang_list` may be undefined, or defined from star imports

(F405)